lib/Expectation.lua

```diff
@@ -197,6 +197,35 @@ function Expectation:equal(otherValue)
 	return self
 end
 
+--[[
+	Assert that our expectation value is equal to another value within some limit, inclusive
+]]
+function Expectation:near(otherValue, limit)
+	assert(type(self.value) == "number", "expectation value should be a number")
+	assert(type(otherValue) == "number", "value should be a number")
+	assert(type(limit) == "number", "limit value should be a number")
```

Comment:
Limit is optional, we should allow it to be nil as well.

Let's use more assertive phrasing for these assertions: `Expectation value must be a number to use 'near'` for the first, `'otherValue' must be a number`, `limit must be a number or nil`.